.github/workflows/pytest_apps.yml

```diff
@@ -40,4 +40,4 @@ jobs:
       env:
         OPENAI_API_KEY: "${{ secrets.OPENAI_API_KEY }}"
       run: |
-        pytest apps/
+        pytest -m "not slow" apps/ examples/
```

Comment:
Should we put them into two jobs?

Reply:
I don't think so. UI apps and console examples are essentioally the same thing: demo for the API